src/app/home/countdown/countdown.component.scss

```diff
@@ -0,0 +1,64 @@
+#countdown-title {
+  color: #64b5f6;
```

Comment:
Avoid hardcoding colors. If you need to use some color, take it from the styles-colors.scss file. You can see an example of this in the navbar.component.scss file.